Previously, if the user of a MongoDB Connector did not have sufficient privileges to perform `find` queries in a `foo` collection, it would be considered disconnected.  This has been fixed.
